data/pages-content/sponsor/information.md

```diff
@@ -7,46 +7,43 @@ subtitle:
 
 <!-- # Deadlines -->
 # Sponsor Information
-<Note>Deadlines and specs of the deliverables will become available later. Stay tuned! If you have any questions, please reach out to the sponsorship team at sponsoring@europython.eu</Note>
 
-<!-- - [Sponsored Talk | Workshop/Tutorial | Poster submission - presenter, presentation title & abstract:](#sponsored-content)
-  June 10
-- [Virtual Swag Page submission:](#virtual-swag-page) June 17
-- [Sponsor Logo submission:](#sponsor-logo) June 17
-- [Technical Case Study Blog Post submission:](#sponsor-blog-post) June 17
+- [Sponsored Talk | Workshop/Tutorial | Poster Submission:](#sponsored-content)
+  June 12
+- [Sponsor Logo Submission:](#sponsor-logo) June 12 (all sponsors)
+- [Tickets Registration & Assignment:](#registration) June 19 (all sponsors)
+- [Booth Customisations:](#booth-customisations) June 19 (Silver and above)
```

Comment:
I would cite the zen of Python here: Explicit is better than implicit.
 I think we should be explicit here and write (Keystone, Diamond, Platinum, Gold, Silver)
 That way if I'm a sponsor, I don't need to keep the order of sponsoring packages in mind.

Reply:
Good point! Added those in https://github.com/EuroPython/website/pull/259/commits/346c8a8083779a7fd961e689b3f14e18b1f772e3 I omitted some of the specifics for the items to be delivered during the conference, e.g. tweets, because it can get a bit too complicated. Happy to implement suggestions though!

Reply:
Thank you! Looks nice